diff --git a/cli.js b/cli.js
index 170637c1713641e196522b1db38ebbeef7a4014b..6935c9ad37c8f313a3a5633055e362f0bddc3c98 100644
--- a/cli.js
+++ b/cli.js
@@ -88,6 +88,9 @@ module.exports = (() => {
             switch ( path.extname(confPath) ) {
                 case '.js':
                     config = require( path.resolve(confPath) ) || {};
+                    if (config.default) {
+                        config = config.default;
+                    }
                     break;
                 case '.json':
                 case '.EXAMPLE':
