diff --git a/cli.js b/cli.js
index 170637c1713641e196522b1db38ebbeef7a4014b..af54afd263fd25f81f7685b3a8ffa79eacf1e24c 100644
--- a/cli.js
+++ b/cli.js
@@ -87,6 +87,9 @@ module.exports = (() => {
         try {
             switch ( path.extname(confPath) ) {
                 case '.js':
+                    if (config.default) {
+                      config = config.default;
+                    }
                     config = require( path.resolve(confPath) ) || {};
                     break;
                 case '.json':
